routes/web.php

```diff
@@ -39,3 +40,22 @@
 //View scenario route.
 Route::get('/stories/{story}/scenarios/{scenario}', [ScenarioController::class, 'show'])->name('scenario.show');
 
+//Route::get('/filter', '\App\Http\Controllers\StoryController@filter');
+Route::get('categories/{category:slug}', function (scopeFilter $category) {
```

Comment:
hebben we deze route echt nodig? filtert ie niet op de standaard index / stories pagina